Meadowlark-js/backends/meadowlark-local-mongodb-opensearch/src/model/Document.ts

```diff
@@ -0,0 +1,40 @@
+// SPDX-License-Identifier: Apache-2.0
```

Comment:
What is the significance of "local" in the name of this backend plugin? Is there anything that is actually restricted to localhost usage, rather than deployment to AWS? I was ideally picturing this package as agnostic to location of MongoDB.